service/arm_templates/security_groups_setup.json

```diff
@@ -100,39 +92,39 @@
           {
             "name": "apiLoadBalancerRule",
             "properties": {
-              "description": "Allow the API load balancer to reach the kubernetes API.",
+              "description": "Allow anyone to reach the kubernetes API.",
               "protocol": "tcp",
               "sourcePortRange": "*",
               "destinationPortRange": "[string(parameters('kubernetesAPISecurePort'))]",
-              "sourceAddressPrefix": "[parameters('apiLoadBalancerCidr')]",
+              "sourceAddressPrefix": "*",
               "destinationAddressPrefix": "[parameters('masterSubnetCidr')]",
               "access": "Allow",
               "direction": "Inbound",
               "priority": "3900"
             }
           },
           {
-            "name": "etcdLoadBalancerRule",
+            "name": "workerSubnetToMasterSubnet",
```

Comment:
This is already allowed by default. 

See https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-nsg

AllowVNetInBound allows ALL traffic within virtual network.

Reply:
As discussed, we override the default Azure rules. Because of that, we need to explicitly open traffic between the subnets.